core/src/main/java/tachyon/worker/WorkerStorage.java

```diff
@@ -322,6 +324,12 @@ public WorkerStorage(InetSocketAddress masterAddress, ExecutorService executorSe
 
     mDataFolder = WorkerConf.get().DATA_FOLDER;
     mUserFolder = CommonUtils.concat(mDataFolder, WorkerConf.USER_TEMP_RELATIVE_FOLDER);
+
+    if (WorkerConf.get().EVICT_STRATEGY_TYPE.needReferenceFrequency()) {
+      mLastAttenuateBlockReferenceMs = System.nanoTime() / 1000000L;
```

Comment:
dcapwell pointed out that milli has clock skew issues with NTP. And the period is configurable so nano is more safe.